code/EventLinkQR/app/src/main/res/layout/create_event_page.xml

```diff
@@ -199,6 +184,21 @@
                 android:textSize="20sp"
                 android:textStyle="bold" />
 
+            <TextView
```

Comment:
why did you move this block

Reply:
In the intent testing, it is essential to have the components displayed on the screen that we are testing, so to keep the intent testing simple without having to scroll the page, I moved geolocation toggle button up (so that it is displayed on the screen while testing)